### ISSUE TITLE
[Bug]: Unable to list newly created credential users

### ISSUE THREAD

**Issue by letian-jiang** [milvus-io/milvus#18117]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2): go-sdk
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

After creating a credential user, we cannot get the username via ListCredUsers.

```go
const (
	addr     = "localhost:19530"
	username = "username"
	password = "password"
)

func main() {
	ctx := context.Background()
	c, err := client.NewGrpcClient(ctx, addr)
	if err != nil {
		log.Fatal("failed to connect:", err)
	}

	users, err := c.ListCredUsers(ctx)
	if err != nil {
		log.Fatal("fail to list credential users:", err)
	}
        fmt.Println(users)

	if err = c.CreateCredential(ctx, username, password); err != nil {
		log.Fatal("failed to create credential:", err)
	}
	time.Sleep(time.Second)

	users, err = c.ListCredUsers(ctx)
	if err != nil {
		log.Fatal("fail to list credential users:", err)
	}
	fmt.Println(users)
	c.Close()
	return
}
```

### Expected Behavior

Get this username via the List interface.

**Comment by aakejiang** [comment]
Fix both in master & version 2.1

/unassign
/assign @letian-jiang @yanliang567

**Comment by letian-jiang** [comment]
Verified.
